com.ibm.wala.cast.python.ml.test/source/com/ibm/wala/cast/python/ml/test/TestTensorflowModel.java

```diff
@@ -65,6 +65,15 @@ public void testTf2() throws ClassHierarchyException, IllegalArgumentException,
 		testTf2("tf2e.py", "add", 2, 2, 3);
 		testTf2("tf2f.py", "add", 2, 2, 3);
 		testTf2("tf2g.py", "add", 2, 2, 3);
+		testTf2("tf2h.py", "add", 2, 2, 3);
+		testTf2("tf2i.py", "add", 2, 2, 3);
+		testTf2("tf2j.py", "add", 2, 2, 3);
+		testTf2("tf2k.py", "add", 2, 2, 3);
+		testTf2("tf2l.py", "add", 2, 2, 3);
+		testTf2("tf2m.py", "add", 2, 2, 3);
+		// TODO: Fix issue #22
+		// testTf2("tf2n.py", "func2", 1, 2);
```

Comment:
Let's add the graph-based code as a test in the meantime, so that we can at least cover `tf.Tensor`. In that Python code, however, please ensure that it is a minimal example, i.e., that there is no unnecessary code.

Reply:
Sounds good.

Reply:
Added it in `tf2p.py`.